packages/components/src/LightBox/LightBox.mdx

```diff
@@ -0,0 +1,95 @@
+---
+name: LightBox
+menu: Components
+route: /components/light-box
+---
+
+import { useState } from "react";
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Button } from "../Button";
+import { LightBox } from ".";
+
+# Light Box
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+LightBox is a component designed to display an array of images to the user. A
+user clicks on a control and is shown a carousel style presentation of the
+defined images.
+
+This component is a wrapper of the frontend-collective's react-image-lightbox
+third party library.
+<a href="https://github.com/frontend-collective/react-image-lightbox">React
+Image Lightbox</a>
```

Comment:
This comment might be better fit for inside the `Lightbox.tsx` file as the consumer of the component doesn't really need to know this. 😄

Reply:
I disagree, I think having it here is useful for devs and designers. There is a lot of out of the box functionality that react-image-lightbox offers that may not be apparent otherwise. If a team is aware that a feature is potentially just a prop away vs. writing that whole feature we could save some work/headaches.